master/internal/configpolicy/postgres_task_config_policy.go

```diff
@@ -88,6 +90,43 @@ func GetTaskConfigPolicies(ctx context.Context,
 	return &tcp, nil
 }
 
+// GetPriorityLimit reads the priority limit for the given scope and workload type.
+// It returns found=false if no limit exists.
+func GetPriorityLimit(ctx context.Context, scope *int, workloadType string) (limit int, found bool, err error) {
+	if !ValidWorkloadType(workloadType) {
+		return 0, false, fmt.Errorf("invalid workload type: %s", workloadType)
+	}
+
+	wkspQuery := wkspIDQuery
+	if scope == nil {
+		wkspQuery = wkspIDGlobalQuery
+	}
+
+	var constraints model.Constraints
+	var constraintsStr string
+	err = db.Bun().NewSelect().
+		Table("task_config_policies").
+		Column("constraints").
+		Where(wkspQuery, scope).
+		Where("workload_type = ?", workloadType).
+		Scan(ctx, &constraintsStr)
+
+	if err == sql.ErrNoRows {
+		return 0, false, nil
+	} else if err != nil {
+		return 0, false, fmt.Errorf("error retrieving priority limit: %w", err)
```

Comment:
I've found that `fmt.Errorf("error retrieving priority limit: %v", err)` prints the error cleaner than `fmt.Errorf("error retrieving priority limit: %w", err)`! I think the `%w` prints the error as a value rather than extracting its string error message